### DISCUSSION TITLE
Can't change stroke width after upgrading from v1.0.4 to current v2.0.18

### DISCUSSION THREAD

**Discussion by ruffjaxson** [tailwindlabs/heroicons#1064] [Help]
I have a React / Typescript app, decided to upgrade our package so we can get the latest icons. 

Using the icon works, but stroke width does not:
`<ChevronRightIcon className="h-5 w-5 text-gray-400" strokeWidth={4} />`

Even changing the css properties in the devtools console on Chrome does nothing. Please help!

**Comment by lucas-0liveira** [comment]
If you're using Heroicons, remove the strokeWidth property and adjust the size directly using the h-5 and w-5 classes.

Example --> <ChevronRightIcon className="h-5 w-5 text-gray-400" />

If you're using icons from another source (like Feather Icons), make sure that the strokeWidth property is supported by the specific icon you're using. If the issue persists, check for any interference in other parts of your code and ensure that you're using the correct version of the icon package. I hope this helps resolve the issue! If you need further assistance, feel free to let me know.
